packages/apps/src/App.tsx

```diff
@@ -13,11 +13,13 @@ import { config as routerConfig } from './router-config';
 
 const appLogger = LoggerService.new('App');
 const App: FC = () => {
+  const {} = useWebContext();
```

Comment:
This seems out of place?

Reply:
The`InteractiveFeedback`is going to be used with errors/info/warning that requires feedback from the user and trigged view via the providers, not just errors, and this is meant to be decoupled from the view.

`notificationApi` is used with this for showing toast messages without feedback although `notificationApi` isn't really decoupled and relay on some react logic not sure if it needs to be decoupled(fully)  as well at least ATM

Reply:
I just mean the return type is `{}`. Is something meant to be returned here?

Reply:
This is a dead code removed it on future commits